docs/src/_guide/advanced-topics.md

```diff
@@ -13,3 +13,159 @@ Most of Meltano's features are available without installing any additional packa
 * `s3` - Support for using S3 as a [state backend](/concepts/state_backends)
 * `gcs` - Support for using Google Cloud Storage as a [state backend](/concepts/state_backends)
 * `azure` - Support for using Azure Blob Storage as a [state backend](/concepts/state_backends)
+
+## Airbyte Connector Integration FAQ
+
+### How do the Singer and Airbyte specifications relate?
+
+The Singer specification was started in 2016 by Stitch Data. It specified a data transfer format that would allow any number of data systems, called taps, to send data to any data destination, called targets. Airbyte was incorporated in 2020 and created their own specification that was heavily inspired by Singer. There are differences but the core of each specification is sending new-line delimited JSON data from STDOUT of a tap to STDIN of a target.
```

Comment:
```suggestion
The Singer specification was started in 2016 by Stitch Data. It specified a data transfer format that would allow any number of data systems, called taps, to send data to any data destinations, called targets. Airbyte was incorporated in 2020 and created their own specification that was heavily inspired by Singer. There are differences, but the core of each specification is sending new-line delimited JSON data from STDOUT of a tap to STDIN of a target.
```